src/components/ButtonTag/ButtonTag.module.scss

```diff
@@ -0,0 +1,65 @@
+@use "src/styles/colors";
+
+$button-height: 3rem;
+$icon-size: 1.5rem;
+$label-size: 0.7rem;
+
+$inner-shadow: inset 0 2px colors.getColor("tertiary", 60);
+
+.buttonTagWrapper {
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    height: fit-content;
+    width: 100%;
+
+    p {
```

Comment:
Dont use element selectors, they affect the whole website

Reply:
in this case I don't think it applies to the whole website, as it is only if it is a child of a buttonTagWrapper